aviary/utils/test_utils/default_subsystems.py

```diff
@@ -0,0 +1,24 @@
+from aviary.subsystems.propulsion.propulsion_builder import CorePropulsionBuilder
+from aviary.subsystems.geometry.geometry_builder import CoreGeometryBuilder
+from aviary.subsystems.mass.mass_builder import CoreMassBuilder
+from aviary.subsystems.aerodynamics.aerodynamics_builder import CoreAerodynamicsBuilder
+from aviary.variable_info.variable_meta_data import _MetaData as BaseMetaData
+from aviary.variable_info.enums import LegacyCode
+
+
+def get_default_premission_subsystems(legacy_code=None, engines=None):
+    legacy_code = LegacyCode(legacy_code)
```

Comment:
Does `LegacyCode(None)` cause problems? If so, should we have a better default or just no default?

Reply:
what about `get_default_premission_subsystems(legacy_code, engines=None)`, so it's clear they need to specify something